README.md

```diff
@@ -0,0 +1,81 @@
+# Covid19 Statistics App
+
+COVID-19 Stats is a single-page app that delivers up-to-date information on COVID-19 in-country, continent, and global total data. The date input filter allows the user to retrieve data on any given date. React, Redux, and Tailwind CSS were used to create the app.
+
+## Screenshots
+
+![react-capston](https://user-images.githubusercontent.com/92645881/163482229-b957948c-4f76-4f29-9b7f-f79bd0ce0cd0.gif)
+
+
+## Built With
+
+- HTML5, CSS3
+- JavaScript ES6, React/Redux, Bootstrap
+- CLI, TDD, gh-pages, Gitflow
+
+## Live demo
+
+[live Demo](covidstatupdate.netlify.app)
+
+## video Presentation Link
+[Loom Link](https://www.loom.com/share/cbda22dcfbe746e79b85620cc764a136)
+
+## Built With
+- `React.js`,
+- `Gitflow`
+- `Redux`
+- `Tailwind CSS`
+- `JavaScript ES6`
+- `HTML5` and `CSS3`
+
+## Technologies used: 
+-  GitHub
+-  Netlify
+
+
+### Setup
+- Open your command prompt or terminal and run.
+- [Clone the GitHub Repository](https://github.com/zmekonnen251/covid19-stats.git)
+- run this commands in your terminal:
+     - `cd covid19-stats`
+     - `npm install`
+     - `npm run build`
+     - `npm start`
+
+### Usage
+- after running `npm start` the website will be opened automatically on your default browser.
+
+### Run tests
+
+- To run tests run : npm run test
+
+
+### Deployment
+
+- Project is deployed using netlify
+
+## Author
+
+👤 **Zelalem Mekonnen**
+
+- GitHub: [GitHub](https://github.com/zmekonnen251)
+- Twitter: [Twitter](https://twitter.com/mek_zela)
+- LinkedIn: [Zelalem G. Mekonnen](https://www.linkedin.com/in/zelalem-getachew/)
+
+
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```

Comment:
- [ ] In `Readme` file there should be one `Acknowledgement` section which gives appropriate credit to[ Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the[ author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the[ Creative Commons License](https://creativecommons.org/licenses/). As requested by the previous reviewer.
- **REQUIREMENT** The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).